r/R/dplyr-datetime-helpers.R

```diff
@@ -176,7 +191,8 @@ build_formats <- function(orders) {
   }
 
   formats_list <- map(orders, build_format_from_order)
-  purrr::flatten_chr(formats_list)
+  formats <- purrr::flatten_chr(formats_list)
```

Comment:
Ooh, I didn't know `purrr` had this function, nice!

Reply:
😉